composer.json

```diff
@@ -20,6 +20,7 @@
         "ext-odbc": "*",
         "ext-json": "*",
         "keboola/db-import-export": "^0.21",
+        "keboola/table-backend-utils": "^0.19",
```

Comment:
Tady to nemusíš dělat ta závislosti je už v import-export, stejně to takto neoveridneš, budeme muset vydat v1 brzo aby to fungovalo dobře.

Reply:
jo, je to pryc